lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -55,7 +55,7 @@
       window.ace_editor = editor;
 
       editor.setTheme("ace/theme/tomorrow");
-      editor.setKeyboardHandler("ace/keyboard/vim");
+      editor.setKeyboardHandler("ace/keyboard/emacs");
```

Comment:
@ProgramFan Could https://github.com/gollum/gollum/pull/1215 be relevant here, so that people can switch between key bindings?